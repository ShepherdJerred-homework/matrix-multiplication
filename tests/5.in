5 0 8 6 3 0 5 3 5 8 0 0 3 9 10 10 9 1 4 4 6 8 3 5 2 5 5 6 4 6 4 10 10 3 1 1 2 1 1 3 3 8 6 3 6 10 5 6 4 7 10  0 1 1 10 10
