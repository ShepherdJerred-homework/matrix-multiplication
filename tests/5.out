44
88
212
105
78
110
33
62
169
180
